package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "fdk-harvest-admin",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "dbup": "docker-compose -f docker-compose-mongo.yml up -d mongodb ",
+    "dbdown": "docker-compose -f docker-compose-mongo.yml stop mongodb ",
+    "start": "NODE_ENV=development ts-node-dev --respawn --no-notify --transpileOnly ./src/server.ts",
+    "build": "rm -rf dist && tsc",
+    "prod": "NODE_ENV=production node ./dist/src/server.js",
+    "lint": "tsc --noEmit && eslint ./src ./test --ext=js,ts --fix",
+    "test": "NODE_ENV=test node node_modules/mocha/bin/_mocha -r ts-node/register test/record.api.test.ts --exit --timeout 10000"
+  },
+  "type": "module",
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "app-root-path": "^2.2.1",
+    "body-parser": "^1.19.0",
+    "config": "^3.2.2",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
+    "express-openapi-validate": "^0.4.4",
+    "js-yaml": "^3.13.1",
+    "keycloak-connect": "^7.0.1",
+    "lodash": "^4.17.15",
+    "mongoose": "^5.7.3",
+    "uuid": "^3.3.3"
+  },
+  "devDependencies": {
```

Comment:
It's a good idea to add useful dev dependencies, such as:
@commitlint/cli @commitlint/config-conventional commitizen cz-conventional-changelog husky lint-staged winston

Reply:
winston is a logging library so it can be a separate feature pull request